packages/perennial/contracts/interfaces/IParamProvider.sol

```diff
@@ -20,10 +20,7 @@ interface IParamProvider {
         uint256 version
     );
 
-    error ParamProviderInvalidMakerFee();
-    error ParamProviderInvalidTakerFee();
-    error ParamProviderInvalidPositionFee();
-    error ParamProviderInvalidFundingFee();
+    error ParamProviderInvalidFeeValue();
```

Comment:
I know these are all fees but might be more consistent to just saying something like invalid param since it's one error for the whole contract here.